sormas-app/app/src/main/res/values/strings.xml

```diff
@@ -236,6 +236,9 @@
     <string name="caption_year">Year</string>
```

Comment:
Most, if not all, of these strings should exist in the API because they are needed in both the web and the mobile app; please use those (or create them if they're missing) instead. Only create strings here that are really ONLY used in the mobile app.